src/imputation/MoR.py

```diff
@@ -112,8 +121,108 @@ def carry_forwards(df, backdata, impute_vars):
         df.loc[match_cond, f"{var}_imputed"] = df.loc[match_cond, f"{var}_prev"]
     df.loc[match_cond, "imp_marker"] = "CF"
 
+    df.loc[match_cond] = create_imp_class_col(df,
+                                              "200_prev",
+                                              "201_prev",
+                                              use_cellno=False)
+
     # Drop merge related columns
-    to_drop = [column for column in df.columns if column.endswith("_prev")]
+    to_drop = [column for column in df.columns
+               if (column.endswith("_prev")) & (column not in impute_vars)]
```

Comment:
good decision to keep the _prev for the impute_vars

Reply:
As you've said in the main comment, can't seem to see them in the imputation qa file - going to track down why this happening now.

Reply:
Have updated this now - weren't selecting the columns to keep properly, but have added some regex to do this.